control/lane_departure_checker/CMakeLists.txt

```
@@ -10,13 +10,13 @@ include_directories(
   ${EIGEN3_INCLUDE_DIRS}
 )
 
-ament_auto_add_library(lane_departure_checker SHARED
+ament_auto_add_library(autoware_lane_departure_checker SHARED
   src/lane_departure_checker_node/lane_departure_checker.cpp
   src/lane_departure_checker_node/lane_departure_checker_node.cpp
 )
 
-rclcpp_components_register_node(lane_departure_checker
-  PLUGIN "lane_departure_checker::LaneDepartureCheckerNode"
+rclcpp_components_register_node(${PROJECT_NAME}
+  PLUGIN "autoware_lane_departure_checker::LaneDepartureCheckerNode"
```

I think there is a small mistake here
```suggestion
  PLUGIN "autoware::lane_departure_checker::LaneDepartureCheckerNode"
```

---

thank you!
addressed in https://github.com/autowarefoundation/autoware.universe/pull/7325/commits/f2ea2daa98b292cc59c180f466808f75daf6d251